tests/MongoDB.Driver.Examples/DocumentationExamples.cs

```diff
@@ -1568,6 +1568,46 @@ public void Index_Example_2()
             result.Should().Be("cuisine_1_name_1");
         }
 
+        [Fact]
+        public void Causal_Consistency_Example_2()
+        {
+            RequireServer.Check().SupportsCausalConsistency();
+
+            this.client.GetDatabase("myDatabase").DropCollection("myCollection");
+
+            // Start Tunable consistency controls example
+            var connectionString = "mongodb://localhost/?readPreference=secondaryPreferred";
+
+            var client = new MongoClient(connectionString);
+            var database = client.GetDatabase("myDatabase");
+            var collection = database.GetCollection<BsonDocument>("myCollection");
+
+            var document = new BsonDocument("name", "Tom");
```

Comment:
I would eliminate this variable. Or at least move it closer to where it is used.